mr/src/test/java/org/apache/iceberg/mr/TestIcebergInputFormats.java

```diff
@@ -337,7 +339,7 @@ public void testLocality() throws Exception {
     helper.appendToTable(null, expectedRecords);
 
     for (InputSplit split : testInputFormat.create(builder.conf()).getSplits()) {
-      Assert.assertArrayEquals(IcebergSplit.ANYWHERE, split.getLocations());
+      Assert.assertArrayEquals(ANYWHERE, split.getLocations());
```

Comment:
What was the problem here?

Reply:
because the IcebergSplit check warning log, so make public 'ANYWHERE' var to private. the TestIcebergInputFormats need define a private internal variable.

```
mr\src\main\java\org\apache\iceberg\mr\mapreduce\IcebergSplit.java:38: [MutablePublicArray] Non-empty arrays are mutable, so this `public static final` array is not a constant and can be modified by clients of this class.  Prefer an ImmutableList, or provide an accessor method that returns a defensive copy.
  public static final String[] ANYWHERE = new String[]{"*"};
                               ^
    (see https://errorprone.info/bugpattern/MutablePublicArray)
```